src/wing/CCPACSWingProfileCST.cpp

```diff
@@ -54,19 +54,6 @@ CCPACSWingProfileCST::~CCPACSWingProfileCST()
 {
 }
 
-std::string CCPACSWingProfileCST::CPACSID()
```

Comment:
there used to be a factory at some point, creating an approproate ITiglWireAlgorithm for a given profile type (CCPACSPointListXYZ vs. CCPACSWingProfileCST) . However, creating the right type is already done by the generator so there was no more need for the factory.